packages/docs-reanimated/docs/scroll/scrollTo.mdx

```diff
@@ -72,6 +72,8 @@ import ScrollToSrc from '!!raw-loader!@site/src/examples/ScrollTo';
 ## Remarks
 
 - The `scrollTo` function can only be called from [the UI thread](/docs/fundamentals/glossary#ui-thread).
+- Supports `Animated.FlatList`.
+- Usually works with other ScrollView-like and FlatList-like components if they use a `ScrollView` under the hood and are [made animated](/docs/core/createAnimatedComponent#reference).
```

Comment:
Let's also mention `getScrollableNode()` method.

Reply:
If we want to describe implementation details, I think that we should also mention `getNativeScrollRef` which is required in the new architecture